docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ycql.md

```diff
@@ -45,7 +45,7 @@ You can restore YugabyteDB universe YCQL data from a backup as follows:
     - **Storage Location**: Specify the storage location.
     - **Universe**: Select the YCQL universe to restore.
     - **Keyspace**: Specify the keyspace.
-    - **Table**: Specify the table to be restored. Notee that the table name must be different than the backed up table name.
+    - **Table**: Specify the table to be restored. Note: The table name must be different than the backed up table name.
```

Comment:
Should be:

Note that the table name must be different than the backed up table name.

Reply:
Changed in latest commit.